source/includes/concept/_dataset.md

```diff
@@ -69,3 +69,28 @@ Data provided in the form of a JSON document.
 #### Extensible (X) Markup Language (XML)
 
 Data provided in the form of a XML document.
+
+
+## Query
+
+In the previous section, we covered the concept of a RW API dataset which, in simple terms, is a way to tell the RW API that your data exists, and where. While cataloging datasets in a public repository is useful, making that data easily accessible is one of the main goals of the RW API. This is where **queries** come in.
```

Comment:
I'd add at the end of this section a link to the query endpoint documentation, to keep things referenced: `/index-rw.html#query`.